app/src/main/res/values/styles.xml

```diff
@@ -39,4 +39,13 @@
     <style name="PopupMenu" parent="@style/Widget.AppCompat.Light.PopupMenu">
         <item name="overlapAnchor">true</item>
     </style>
+
+    <style name="ContextMenuItemStyle" parent="@style/AppTheme">
+        <!-- Set the padding to the children (menu item) of the NavigationView
```

Comment:
@mcomella  After some investigation i found that Items layout in **NavigationView** are defined in [design_navigation_item.xml](https://android.googlesource.com/platform/frameworks/support.git/+/master/design/res/layout/design_navigation_item.xml). 
```
<android.support.design.internal.NavigationMenuItemView
        xmlns:android="http://schemas.android.com/apk/res/android"
        android:layout_width="match_parent"
        android:layout_height="?attr/listPreferredItemHeightSmall"
        android:paddingLeft="?attr/listPreferredItemPaddingLeft"
        android:paddingRight="?attr/listPreferredItemPaddingRight"
        android:foreground="?attr/selectableItemBackground"
        android:focusable="true"/>
```

As you can see some properties are set using attributes that are specific to themes. So in order to use a custom value for a property, we need to override its related attribute.

> t'd be also great to comment that PaddingStart/End don't work in this context and a reason, if you know why (I think it's because only L/R is defined in AppCompat, whose theme we inherit from; only the system defines S/E).

I think it's just because ListPreferredItemPaddingStart and istPreferredItemPaddingEnd are not used in the source.